src/models/stream.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Stream extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Stream.init(
+    {
+      name: DataTypes.STRING,
+    },
+    {
+      sequelize,
+      modelName: 'Stream',
+    }
+  );
+  Stream.associate = function (models) {
+    Stream.hasMany(models.Student, { as: 'students' });
+  };
```

Comment:
Думаю лучше так сделать
```suggestion
    static associate(models) {
    this.hasMany(models.Student);
    }
  }
  Stream.init(
    {
      name: DataTypes.STRING,
    },
    {
      sequelize,
      modelName: 'Stream',
    }
  );

```

Reply:
переместил